docs/dapp/sapphire/quickstart.mdx

```diff
@@ -336,6 +95,12 @@ any network config entry corresponding to the Sapphire's chain ID will
 automatically be wrapped with Sapphire specifics for encrypting and signing the
 transactions.
 
+#### Get some Sapphire Testnet tokens
+
+Now for the fun part. We need to configure the Sapphire network and get some tokens.
+Hit up the one and only [Oasis Testnet faucet] and this time select "Sapphire".
```

Comment:
1. [Oasis Testnet faucet] has no link.
2. Also, can you put `any network config entry corresponding to the Sapphire's chain ID will
automatically be wrapped with Sapphire specifics for encrypting and signing the
transactions` in bold?

Reply:
🤐 hvala